### DISCUSSION TITLE
Consider adding commons imaging ?

### DISCUSSION THREAD

**Discussion by AlexisDrogoul** [gama-platform/gama#46] [Ideas]
Given the difficulty we have to read some Tiff images, for instance, I wonder if using https://github.com/apache/commons-imaging would not be a good idea. It seems pretty solid on the type of images we're reading (more than JAI/ImageIO).

**Comment by lesquoyb** [comment]
How has the handling of images evolved since then ?
Do you think it's still a good idea to use that library ? 
I would think so, at least to try, as it is still maintained and I know some image operators are pretty slow (see my incredible [fps game](https://github.com/lesquoyb/gama.wolfenstein/tree/main) that is super laggy but shouldn't, and all of the lag seems to come from the `with_size` operator on images)
